### DISCUSSION TITLE
Dark Mode / Monochrome Icons

### DISCUSSION THREAD

**Discussion by Hate9** [Floorp-Projects/Floorp#1257] [Feature Request]
I've noticed that the Floorp icon shows up very poorly against dark colors. Would it be possible to have an option for a monochrome or alternate-color icon for desktops and browsers in dark mode?
![image](https://github.com/Floorp-Projects/Floorp/assets/4684153/10a83e93-8019-4173-9741-2d16e4e9b933)
![image](https://github.com/Floorp-Projects/Floorp/assets/4684153/0bfb798d-b6cb-4a84-99c4-edc73b218761)
(image 1 is inside floorp, image 2 is the icon on my desktop's window frames)

**Comment by mathiaspl20** [comment]
I would love to have an option for a brighter and perhaps more colorful icon.

While I don't mind the icon generally, I also have a hard time seeing it against a dark background.

Even just having the icon colors inverted would work well in my opinion

**Comment by CutterKnife** [comment]
Monochrome icons already exist. All that is needed is to apply it elsewhere.
